src/common/GlobalStyles.js

```diff
@@ -14,6 +14,7 @@ html {
 body {
     font-family: 'Poppins', sans-serif;
     font-weight: 400;
+    padding: 16px;
```

Comment:
When you delete the padding then you delete this little space around the image with the title:
![image](https://user-images.githubusercontent.com/123471023/233119457-cffd5b40-6a94-4615-836b-52e3cbdf1d32.png)

Reply:
See?
![image](https://user-images.githubusercontent.com/123471023/233123061-f2d78e5f-d801-4b2d-913b-01667350594b.png)